spotinst/resource_spotinst_ocean_spark.go

```diff
@@ -195,27 +198,81 @@ func resourceSpotinstSparkClusterDelete(ctx context.Context, resourceData *schem
 		return diag.FromErr(err)
 	}
 
-	log.Printf("===> Cluster deleted successfully: %s <===", resourceData.Id())
+	log.Printf("===> cluster deleted successfully: %s <===", resourceData.Id())
```

Comment:
```suggestion
	log.Printf("===> Cluster deleted successfully: %s <===", resourceData.Id())
```

I think we should keep the log strings capitalized for consitency